sklearn/preprocessing/tests/test_data.py

```diff
@@ -1217,7 +1218,7 @@ def test_normalize():
 
 
 def test_binarizer():
-    X_ = np.array([[1, 0, 5], [2, 3, -1]])
+    X_ = np.array([[1, 0, 5], [2, 3, -1]], dtype=np.float64)
```

Comment:
I get this failure which is a result of being unable to copy... 

```
$ nosetests -v preprocessing/tests/test_data.py:test_binarizer

sklearn.preprocessing.tests.test_data.test_binarizer ... FAIL

======================================================================
FAIL: sklearn.preprocessing.tests.test_data.test_binarizer
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/scikit-learn/sklearn/preprocessing/tests/test_data.py", line 1250, in test_binarizer
    assert_true(X_bin is X)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 1 test in 0.002s

FAILED (failures=1)
```

and yes changing it to `X_bin is not X` will also fix this... In that case I'll also add a test which succeeds for np.float64 alone... thanks!